vtexdocs/help-center-content
# feat: generate redirects

[EDU-13140](https://vtex-dev.atlassian.net/browse/EDU-13140)

created a script that generates a netlify.toml file with:

- redirects from legacy help center content to new content paths. it covers the following scenarios:
   - complete URL (with locale, slug, and ID)
   - only locale and ID in the URL
   - only ID in the URL
   - only locale and slug in the URL
   - only slug in the URL
   - only legacy slug in the URL

- redirects exported from https://vtexhelp.myvtex.com/admin/cms/redirects

steps for testing:
1. run `node docs-utils/generateRedirects.js`.
2. check the content of the newly generated netlify.toml file in the root folder.
3. follow instructions in https://github.com/vtexdocs/helpcenter/pull/128
